user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -160,33 +162,39 @@ There are two suite hook types available:
 * ``@BeforeSuite`` - executed before any feature runs.
 * ``@AfterSuite`` - executed after all features have run.
 
-Feature Hooks
--------------
+Tagged Hooks
```

Comment:
Tagged hooks are not different kind of hook. Tagging hooks is an additional feature support ed by some of the hook types described in this chapter.

Btw, the title level should not be at the same level than the different hook types. It should be a different section of the chapter compared to the list of available hooks.